src/Education/Commands/RuleDescriptionUserControl.xaml

```diff
@@ -1,10 +1,123 @@
+            <Setter Property="Background" Value="{DynamicResource {x:Static vsshell:VsBrushes.InfoBackgroundKey}}" />
+            <Setter Property="Padding" Value="8,5,5,8" />
+        </Style>
+        <Style x:Key="UnorderedList" TargetType="List">
+            <Setter Property="MarkerStyle" Value="Circle" />
+            <Setter Property="Margin" Value="2" />
+        </Style>
+        <Style x:Key="OrderedList" TargetType="List">
+            <Setter Property="MarkerStyle" Value="Decimal" />
+            <Setter Property="Margin" Value="2" />
+        </Style>
+        <Style x:Key="Code_Span" TargetType="Span">
+            <Setter Property="Foreground" Value="{DynamicResource {x:Static vsshell:VsBrushes.InfoTextKey}}"/>
+            <Setter Property="Background" Value="{DynamicResource {x:Static vsshell:VsBrushes.InfoBackgroundKey}}" />
+            <Setter Property="FontFamily" Value="Courier New" />
+            <Setter Property="FontSize" Value="{DynamicResource {x:Static vsshell:VsFonts.Environment90PercentFontSizeKey}}"/>
+        </Style>
+    </UserControl.Resources>
+
+
+    <FlowDocumentScrollViewer x:Name="docViewer">
+        <FlowDocument >
+            <Paragraph Style="{DynamicResource Heading2_Paragraph}">SonarLint rules help</Paragraph>
+            <Paragraph>
+                For each Sonar rule, we provide code samples and offer guidance on a fix. Even more importantly, we also tell you why.
+            </Paragraph>
+            <Paragraph>
+                To see the help for a specific Sonar rule, view the issue in the
+                    <Span Style="{DynamicResource Code_Span}" >Error List</Span>, then either:
+
+            </Paragraph>
+            <List MarkerStyle="Circle">
+                <ListItem>
+                    <Paragraph>Click on the hyperlink in the
+                        <Span Style="{DynamicResource Code_Span}">Code</Span> column
+                    </Paragraph>
+                </ListItem>
+            </List>
+            <Paragraph>or</Paragraph>
+            <List MarkerStyle="Circle">
+                <ListItem>
+                    <Paragraph>Select the appropriate row in the
+                        <Span Style="{DynamicResource Code_Span}">Error List</Span> column, then
+                    </Paragraph>
+                </ListItem>
+                <ListItem>
+                    <Paragraph>Select
+                        <Span Style="{DynamicResource Code_Span}">Show Error Help</Span> from the context menu
+                    </Paragraph>
+                </ListItem>
+            </List>
+            <Paragraph>The rule help will be shown in this tool window.</Paragraph>
+            <Paragraph>By default, this tool window is docked with the
+                <Italic>Solution Explorer</Italic>. However, you can move it any where you like, just like any other tool window.
+            </Paragraph>
+        </FlowDocument>
+    </FlowDocumentScrollViewer>
```

Comment:
Boilerplate text that appears before the user selects help for a particular rule.

![image](https://user-images.githubusercontent.com/10757559/216384566-be5b0879-ca13-4e0d-9c65-018863d39d69.png)

Reply:
I like it! It actually explains to users how to get the rule descriptions which is good!